applications/FluidDynamicsApplication/custom_conditions/stokes_wall_condition.cpp

```diff
@@ -129,6 +177,23 @@ void StokesWallCondition<3,3>::CalculateNormal(array_1d<double,3>& An )
     An *= 0.5;
 }
 
+template <>
+void StokesWallCondition<3,4>::CalculateNormal(array_1d<double,3>& An )
```

Comment:
I have implemented this method to compute Normal (times Area) for quad.

Reply:
this should already be implemented in the Geometry itself, can you check please?

Reply:
yes, Normal function is in Geometry. Removed this function.